autonomy/cli/scaffold_fsm_templates/tests.py

```diff
@@ -62,7 +62,7 @@ class RoundTestCase:
 
         name: str
         initial_data: Dict[str, Hashable]
-        payloads: BaseTxPayload
+        payloads: Mapping[str, BaseTxPayload]
```

Comment:
This change is what is breaking the tests

Reply:
That was caused by a missing import in the template string, I've fixed it here https://github.com/valory-xyz/open-autonomy/pull/1446/commits/7a1f2a48bd0ab76b60e032cec55ec96e877116cb